src/helpers/dbServer.js

```diff
@@ -0,0 +1,24 @@
+const { MongoMemoryServer } = require("mongodb-memory-server");
+const mongoose = require("mongoose");
+
+let db = undefined;
```

Comment:
You can skip adding 'undefined' by default to the db variable.

Reply:
Typo or autocomplete involved. Fixed in the recent PR.

Reply:
Closing this one.